app-web/src/components/Cards/Card/Card.js

```diff
@@ -45,7 +50,31 @@ const Card = ({ type, title, description, image, link, ...rest }) => {
     </CardDescription>
   );
 
-  if (image && description) {
+  //first check if its an eventbrite event, as in our resolver in gatsby-node.js -> image is set to "eventbrite"
+  if (image === 'eventbrite' && description) {
+    cardBody = (
+      <Aux>
+        <CardDescription title={description} clamp={3} tagName="p">
```

Comment:
Instead of making the card component smarter, (it has to keep track of event brite events...and than meetups, and then yammer) let's make it dumber. There is the `renderProps` feature in react which may work well for this. If there are render props, don't render a card body, just render the `renderProps`.